pkg/permissions/set.go

```diff
@@ -137,3 +138,25 @@ func (ps *Set) IsSubSetOf(parent Set) bool {
 
 	return true
 }
+
+// HasSameRules returns true if the two sets have exactly the same rules.
+func (ps Set) HasSameRules(other Set) bool {
```

Comment:
This would be confusing, we already have `IsSubSetOf` which also check rules content : `io.cozy.files:GET:some-id < io.cozy.files`